### DISCUSSION TITLE
Add Character to End of Line

### DISCUSSION THREAD

**Discussion by cloud303-cholden** [zed-industries/zed#16043] [Show and tell]
One feature I've been missing coming from Neovim is from [this plugin](https://github.com/saifulapm/chartoggle.nvim) which toggles a `,` or `;` to the end of the line. I figured out the Zed recipe for this. It doesn't toggle, but hopefully someone will find it useful. I used [this](https://stackoverflow.com/a/45164592) as a reference for storing the cursor position.

Here are the keymaps:
```json
  {
    "context": "Editor",
    "bindings": {
      "alt-,": [
        "workspace::SendKeystrokes",
        "escape m s shift-h m t ' t z t ` s shift-a , escape ' t z t ` s"
      ],
      "alt-;": [
        "workspace::SendKeystrokes",
        "escape m s shift-h m t ' t z t ` s shift-a ; escape ' t z t ` s"
      ]
    }
  }
```

**Comment by supersuryaansh** [comment]
For me Zed is automatically adding a semicolon as soon as I press save when working with Nodejs, is there a way to stop this from happening?
